packages/scripts/src/commands/app-data/convert/utils/app-data-string.utils.ts

```diff
@@ -122,3 +122,8 @@ export function extractDynamicDependencies(dynamicFields: FlowTypes.TemplateRow[
   });
   return dynamicDependencies;
 }
+
+// Standardised newline characters within a string (i.e. replace "\r\n" and "\r" with "\n")
```

Comment:
I haven't found any cases of `\r` not followed by `\n` . Do you expect these will show up at some point and if so are you sure we want to replace these?

Reply:
Agreed, `\r` doesn't exist in the wild without `\n` (see my [comment](https://github.com/IDEMSInternational/parenting-app-ui/issues/1838#issuecomment-1469751567) on the linked issue), so just need the single replacement